repo2docker/buildpacks/conda/install-base-env.bash

```diff
@@ -23,6 +23,8 @@ chmod 0755 "$MICROMAMBA_EXE"
 
 eval "$(${MICROMAMBA_EXE} shell hook -p ${CONDA_DIR} -s posix)"
```

Comment:
Maybe try using `-r` instead of `-p` when `shell...` is used.

Reply:
I forgot to specify the shell, and I think this caused the error.

Reply:
The prefix path is missing since `-r` is not used indeed.

Done with 6ade75f.